src/consts.rs

```diff
@@ -173,90 +171,6 @@ impl PartialOrd for Constant {
     }
 }
 
-fn format_byte(fmt: &mut fmt::Formatter, b: u8) -> fmt::Result {
-    if b == b'\\' {
-        write!(fmt, "\\\\")
-    } else if 0x20 <= b && b <= 0x7e {
-        write!(fmt, "{}", char::from_u32(b as u32).expect("all u8 are valid char"))
-    } else if b == 0x0a {
-        write!(fmt, "\\n")
-    } else if b == 0x0d {
-        write!(fmt, "\\r")
-    } else {
-        write!(fmt, "\\x{:02x}", b)
-    }
-}
-
-impl fmt::Display for Constant {
```

Comment:
It was used only once, only to print integers. After I refactored the loop lint to use `const_eval` it wasn’t used anymore and seemed of little use, if the user wrote `6*7` in an expression, they must have a reason not to have written `42`, a snippet should always be used to print the original expression.